DOCUMENTATION.md

```diff
@@ -50,3 +50,9 @@ Various Types used in this package can also be imported:
 ```ts
 import * as Types from '@oneblink/sdk/types'
 ```
+
+## OneBlink Role Permissions
+
+Certain actions in the OneBlink SDK require that your key has particular permissions.
+These permissions can be set in the [OneBlink Console](console.oneblink.io). If you do not have access to the OneBlink console, you will need to work with your Administrator to associate the permissions that you require with your key.
```

Comment:
we should not be linking to the oneblink console here but instead be referencing the _tenants console_